website/server/libs/challenges/index.js

```diff
@@ -74,6 +74,9 @@ export async function createChallenge (user, req, res) {
     if (groupBalance >= prizeCost) {
       // Group pays for all of prize
       group.balance -= prizeCost;
+
+      await user.updateBalance(0, 'create_challenge', challenge._id, `Challenge Name: ${challenge.name} (ID: ${challenge._id}) - 
+Balance removed from Group: ${group._id}`);
```

Comment:
@Tressley @saraolson  do we want to have `Balance removed from Group` Information in the History ?

Currently this information is added to the `referenceText` and with that it would be listed with the challenge name combined. This would also end up with being combined in the Challenge Name Link and with that I might not look good. 

Or I could create an property to store such additional informations in - so that we'd still have all the possibilities in styling it the way we want

Reply:
Would `Balance removed from Group` be when a challenge is created with a guild's gems?

You can see here that I've provided examples for the various history entries, as well as their associated strings: https://zpl.io/Pq5qO9L